desktop/src/styles.scss

```
@@ -0,0 +1,2176 @@
+@import 'tailwindcss';
+
+/* ══════════════════════════════════════════════════════════════════════════
+   CheckAI Desktop · Enterprise Design System
+   ══════════════════════════════════════════════════════════════════════════ */
+
+/* ── Tailwind v4 Theme Extension ───────────────────────────────────────── */
+@theme {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)